vertx-web-client/src/main/java/io/vertx/ext/web/client/CachingWebClient.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
```

Comment:
@vietj is this the correct form of header? The examples reference either putting my name in or a generic "The authors" statement.

Reply:
you can use the header that gives copyright to the "Contributors to the Eclipse Foundation", look at `WebClientSession`